src/memgraph.cpp

```diff
@@ -424,6 +430,45 @@ void InitializeLogger() {
   CreateLoggerFromSink(sinks, ParseLogLevel());
 }
 
+std::pair<std::string, std::string> LoadUsernameAndPassword(std::string pass_file) {
+  std::ifstream file(pass_file);
+  if (file.fail()) {
+    spdlog::warn("Problem with opening MG_PASSFILE, memgraph server will start without user");
+    return {"", ""};
+  }
+  std::vector<std::string> result;
+
+  std::string line;
+  std::getline(file, line);
+  size_t pos = 0;
+  std::string token;
+  std::string delimiter = ":";
+  while ((pos = line.find(delimiter)) != std::string::npos) {
+    if (line[pos - 1] == '\\') {
+      line.erase(pos - 1, 1);
+      token += line.substr(0, pos);
+      line.erase(0, pos);
+
+    } else {
+      token += line.substr(0, pos);
+      result.push_back(token);
+      line.erase(0, pos + delimiter.length());
+      token = "";
+    }
```

Comment:
Can you share the format of the pass file?

Reply:
The pass file has the format `username:password`.